src/initial_conditions/initial_conditions.jl

```diff
@@ -1177,7 +1177,7 @@ end
 function gcm_initial_conditions(external_forcing_file)
     NC.NCDataset(external_forcing_file) do ds
         (  # TODO: Cast to CuVector for GPU compatibility
-            gcm_driven_profile(ds, "thetali_mean")[:, 1],  # 1 is initial time index
+            gcm_driven_profile(ds, "theta_mean")[:, 1],
```

Comment:
I think `thetali_mean` is more correct here, because you are using `PhaseEquil_ρθq` to construct the initial condition thermo state, where θ is liquid ice potential temperature.